pkg/model/kv.go

```diff
@@ -17,6 +17,15 @@
 
 package model
 
+//KVRequest is http request body
+type KVRequest struct {
+	Key       string            `json:"key" yaml:"key"`
+	Value     string            `json:"value,omitempty" yaml:"value,omitempty"`
+	ValueType string            `json:"value_type,omitempty" bson:"value_type,omitempty" yaml:"value_type,omitempty"` //ini,json,text,yaml,properties
+	Checker   string            `json:"check,omitempty" yaml:"check,omitempty"`                                       //python script
```

Comment:
Why use python script?

Reply:
it is a TODO, this python scrript will be a function which validate the value